src/libraries/Microsoft.Extensions.Options/src/ValidateOptionsResultBuilder.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Options
+{
+    /// <summary>
+    /// Builds <see cref="ValidateOptionsResult"/> with support for multiple error messages.
+    /// </summary>
+    [DebuggerDisplay("{_errors?.Count} errors")]
```

Comment:
Last I knew, it was generally important to keep DebuggerDisplay syntax trivial, as it would get invoked by the debugger using the language currently being used; this would presumably fail, for example, if used in a VB project.  Instead one typically defines a private property, e.g.
```C#
private int ErrorsCount => _errors?.Count;
```
and then references that:
```C#
[DebuggerDisplay("{ErrorsCount} errors")]
```

Reply:
I'll apply your suggestion. Thanks!